recipes/amplicontyper/meta.yaml

```diff
@@ -1,26 +1,30 @@
-{% set name = "HandyReadGenotyper" %}
-{% set version = "0.1.24" %}
+{% set name = "AmpliconTyper" %}
+{% set version = "0.1.27" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/AntonS-bio/HandyReadGenotyper/archive/{{ version }}.tar.gz
-  sha256: 11970c9bff8b4b7569e7a124030ccc9a0c53a70bb55fe49fcda735857781a731
+  url: https://github.com/AntonS-bio/AmpliconTyper/archive/{{ version }}.tar.gz
+  sha256: e25b71eca63e62c56baffa0a447872aa48b1327ece16e6463ed3527d5389928a
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   run_exports:
-      - {{ pin_subpackage('handyreadgenotyper', max_pin="x.x") }}  
+      - {{ pin_subpackage('amplicontyper', max_pin="x.x") }}  
   number: 0
 
 requirements:
+  build:
+    - setuptools
```

Comment:
Are you sure this is needed?

Reply:
Oddly yes, I didn't have it in previous releases and "conda-build" worked fine, but not this time. I've tried a lot of things and this is the only one removed the "module not found error",

Reply:
I think the one in `host` should be enough.
Let me build without the one in `build` and if it fails I will revert it!

Reply:
You are right, the one in host is enough. I've updated meta.yaml for future releases. Thanks for checking.